css/revops.css

```diff
@@ -144,17 +163,35 @@ p {
   margin-bottom: 1.5em;
 }
 
+.h2-wrapper {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+}
+
+.h2-icon {
+  background: #1b2b3a;
+  background-image: linear-gradient(40deg, #1b2b3a 30%, #455d75 90%);
+  border-radius: 100%;
+  margin-right: 15px;
+  height: 60px;
+  width: 60px;
+  padding: 8px;
+}
+
 .section-heading {
```

Comment:
As this isn't our general section styling i would call it `revops-section__heading` to avoid confusion.

Reply:
fixed